web/html/src/components/datetime/DateTime.tsx

```diff
@@ -3,7 +3,7 @@ import { localizedMoment } from "utils";
 import { getTimeProps } from "./timeProps";
 
 type Props = {
-  value?: string | moment.Moment;
+  value?: string | moment.Moment | Date;
```

Comment:
We would very much like to avoid this, if at all possible. The reason is timezones, when we start passing around native Date objects we will very quickly struggle to make any sense of what the related offsets should be.  

I couldn't find the diff what we need this for, perhaps I just missed it?

Reply:
Ok, no problem, I change it because I saw `<FromNow />` declares it in this way, I thought it was just forgotten :sweat_smile:

Reply:
I'm using this in the `HubDetailsForm` by the way, There I declare both a `<FromNow />` and a `<HumanDateTime />` trying to use a `Date` field. The former works, the latter rejects the type. That's why I changed.

Reply:
Ah okay, I see, so these are values we are sending over from the server, and you want to pass those in? In that case I think it's fine, we can go ahead with this change. Essentially we want to avoid any frontend code that uses `new Date()` and such, and we have a linter rule for it as well, but forbidding it in many places makes it more clear that it shouldn't be done. But for receiving backend values it's fine.

Reply:
Yes, they come from the server JSON response as dates:
```typescript
export type HubDetailData = {
  id: number;
  fqdn: string;
  rootCA: string | null;
  gpgKey: string | null;
  sccUsername: string;
  created: Date;
  modified: Date;
};
```
I can also convert them to `Moment`  before passing it to the component, though.

Reply:
I think for values we receive from the server it's not a problem per se because there we know what we're working with. The problem is the frontend where if you use `Date` and your computer timezone is different than what your server or user configuration is, then there will be an issue.